### PR TITLE
Solved #19.js reverseArr

### PR DESCRIPTION
**Pull request by bramakrishna16**

### CODE REVIEW

File: test/19.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  if(compareArr(arr1,result) == true){
+    console.log(`Right! test case passed`);
+  }else{
+    console.log(`Wrong!!`);
+  }
+}
+
+tester([1,2,3,4],[4,3,2,1]);
+tester([], []);
+tester([1,3,5,7,9], [9,7,5,3,1]);
```
Comment (evo777) [inline test/19.js]: Maybe have some tests that fail?

Comment (songz) [reply test/19.js]: all test must pass. This way, in a work setting if someone changes and breaks your algorithm, the tests you created will start failing and that person would know he broke your code.

File: test/19.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+const reverseArr = require ('../solutions/19.js');
+
+const compareArr = (arr1,result,i=0) => {
```
Comment (songz) [inline test/19.js]: I notice you use compareArr in multiple places. You should plan to create a helper file so you don't have to write this over n over again.

File: test/19.js (hunk 1 above)
Comment (evo777) [reply test/19.js]: Okay, sound good.

**Pull request merged**